api/v1beta1/common.go

```diff
@@ -63,6 +91,44 @@ type NodeSection struct {
 	// <https://kubernetes.io/docs/concepts/configuration/secret/#ssh-authentication-secrets>
 	AnsibleSSHPrivateKeySecret string `json:"ansibleSSHPrivateKeySecret,omitempty"`
```

Comment:
this is already under AnsibleOpts. I don't think it's needed here as well.

Reply:
Done